src/Service/Resolvers/Sql Query Structures/SqlDeleteQueryStructure.cs

```diff
@@ -53,7 +53,7 @@ public class SqlDeleteStructure : BaseSqlQueryStructure
                     Predicates.Add(new Predicate(
                         new PredicateOperand(new Column(DatabaseObject.SchemaName, DatabaseObject.Name, backingColumn!)),
                         PredicateOperation.Equal,
-                        new PredicateOperand($"{MakeParamWithValue(GetParamAsSystemType(param.Value.ToString()!, backingColumn!, GetColumnSystemType(backingColumn!)))}")
+                        new PredicateOperand($"{MakeDbConnectionParam(GetParamAsSystemType(param.Value.ToString()!, backingColumn!, GetColumnSystemType(backingColumn!)), backingColumn)}")
```

Comment:
>backingColumn [](http://example.com/codeflow?start=170&length=13)

`backingColumn!` for consistency

Reply:
We don't need to use ! operator again if we have used it once on a variable. We can even remove the other ! operator used on the previous reference.